servicetalk-http-api/src/main/java/io/servicetalk/http/api/DefaultHttpHeaders.java

```diff
@@ -58,17 +58,6 @@
         this.validateCookies = validateCookies;
     }
 
-    @Override
-    MultiMapEntry<CharSequence, CharSequence> newEntry(final CharSequence key,
-                                                       final CharSequence value, final int keyHash) {
-        return new MultiMapEntry<CharSequence, CharSequence>(value, keyHash) {
```

Comment:
I think at one point this class may have been used in multiple places, however this is no longer the case so I consolidated the key into the base class.